SUPPORTED-FORMATS.md

```diff
@@ -417,6 +403,79 @@ If your tool is supported, but some properties are missing (icon, URL, etc.), pl
                 -
             </td>
         </tr>
+        <td colspan="4">
+            :bulb: run 
+                <pre><code>
+steps{
+    sh 'CodeChecker analyze compilecommands.json --output dir_to_write_plist_files_into'
+    sh 'CodeChecker parse dir_to_write_plist_files_into > report.txt'
+    recordIssues tool:analysisParser(id: 'code-checker', pattern: 'report.txt')
+}
+</code></pre>
+        </td>
+        <tr>
+            <td>
+                code-checker-clang-static-analysis
+            </td>
+            <td>
+                -
+            </td>
+            <td>
+                CodeCheckerClangStaticAnalysis
+            </td>
+            <td>
+                -
+            </td>
+        </tr>
+        <td colspan="4">
+            :bulb: CodeChecker can run clang-tidy and clang static analysis in one run and
+put all warnings in the same file.
+Or the user can choose to run either clang-tidy or clang static analysis.
+
+Both are different analysis types with different scopes and
+likelihood of false positives.
+This name alias enables warnings-ng-plugin to parse the same CodeChecker report.txt as a distinct tool. 
+            <p>
+                <pre><code>
+steps{
+    sh 'CodeChecker analyze compilecommands.json --analyzers clangsa --output dir_to_write_plist_files_into'
+    sh 'CodeChecker parse dir_to_write_plist_files_into > report.txt'
+    recordIssues tool:analysisParser(id: 'code-checker-clang-static-analysis', pattern: 'report.txt')
+}
+                </code></pre>
+            </p>
+        </td>
+        <tr>
+            <td>
+                code-checker-clang-tidy
+            </td>
+            <td>
+                -
+            </td>
+            <td>
+                CodeCheckerClangTidy
+            </td>
+            <td>
+                -
+            </td>
+        </tr>
+<td colspan="4">
+:bulb: CodeChecker can run clang-tidy and clang static analysis in one run and
+put all warnings in the same file.
+Or the user can choose to run either clang-tidy or clang static analysis.
+
+Both are different analysis types with different scopes and
+likelihood of false positives.
+This name alias enables warnings-ng-plugin to parse the same CodeChecker report.txt as a distinct tool.
+<p>
+<pre><code>
+steps{
+    sh 'CodeChecker analyze compilecommands.json --analyzers clang-tidy --output dir_to_write_plist_files_into'
+    sh 'CodeChecker parse dir_to_write_plist_files_into > report.txt'
+    recordIssues tool:analysisParser(id: 'code-checker-clang-tidy', pattern: 'report.txt')
```

Comment:
The documentation must not use Jenkins internals since this library is used by Github Actions as well.